lib/game.rb

```diff
@@ -0,0 +1,49 @@
+class Game
+  attr_reader :players, :turns, :winner
+
+  def initialize
+    @players = {}
+    @board = Board.new
+    @winner = false
+    @turns = 1
+  end
+
+  def add_player(name, token)
+    # max. number of players is 2
+    return false if @players.length >= 2
+
+    players[name] = Player.new(name, token)
+  end
+
+  def show_board
+    p 'This is the current board:'
+    @board.show_board
+  end
```

Comment:
In line 19 you have one `p`. Remember that the print/puts must be only on `main.rb`